### ISSUE TITLE
Is possible to come back from wexBim to IFC?

### ISSUE THREAD

**Issue by boggy94** [xBimTeam/XbimExchange#38]
I'm looking to edit the wexbim file online(webpage based) and export it again to an IFC readable format.

**Comment by andyward** [comment]
No, because Wexbim is only a **geometry** format. It contains no meaningful semantic data, only an encoded form of the following data: the entity type, the entity label, a mesh & styles. 

You should be able to convert the geometry back into some form of IfcRepresentation but you'd need to correlate it with some semantic data to make a useful IFC. The Cobie* formats might be a useful starting point for this.
